fotoobo/cli/fgt/main.py

```diff
@@ -36,10 +36,16 @@ def callback(context: typer.Context) -> None:
 def backup(
     host: str = typer.Argument(
         "",
-        help="The FortiGate hostname to access (must be defined in inventory)",
+        help="The FortiGate(s) to backup (must be defined in inventory)",
```

Comment:
Change the help text, that it is clear that all defined fortigates are meant when "host" is left empty